modules/swagger-annotations/src/main/java/io/swagger/v3/oas/annotations/info/License.java

```diff
@@ -9,7 +10,8 @@
 /**
  * The annotation may be used in {@link Info#license()} to define a license for the OpenAPI spec.
  *
- * @see <a target="_new" href="https://github.com/OAI/OpenAPI-Specification/blob/3.0.1/versions/3.0.1.md#licenseObject">License (OpenAPI specification)</a>
+ * @see <a target="_new" href="https://github.com/OAI/OpenAPI-Specification/blob/3.0.1/versions/3.0.1.md#licenseObject">License (OpenAPI 3.0 pecification)</a>
+ * @see <a target="_new" href="https://github.com/OAI/OpenAPI-Specification/blob/3.0.1/versions/3.1.0.md#licenseObject">License (OpenAPI 3.1 specification)</a>
```

Comment:
Link is not correct as pointing to 3.0.1 branch for 3.1.0 spec. Here and in other changes as well